repository/q.json

```diff
@@ -359,6 +359,18 @@
 				}
 			]
 		},
+		{
+			"name": "QuickTemplate (Go)",
+			"details": "https://github.com/SharanSharathi/sublime_qtpl",
+			"issues": "https://github.com/SharanSharathi/sublime_qtpl/issues",
```

Comment:
The issues url is the default url in GitHub so you don't need to (and therefore shouldn't) provide it here.

Reply:
Thanks for the feedback @braver , I've removed the issues url.